src/state/electrumX/actions.ts

```diff
@@ -0,0 +1,23 @@
+export enum ElectrumXAction {
+  StartListeners = 'StartListeners',
+  SetBlockHeight = 'SetBlockHeight',
```

Comment:
why do we need it in redux?

Reply:
<img width="932" alt="Zrzut ekranu 2020-10-7 o 12 55 53" src="https://user-images.githubusercontent.com/67273035/95322232-783b3a80-089c-11eb-8107-8bc9d453492c.png">
this way we can update transactions confirmations in real time with the usage of electrumXSelectors.blockHeight